### PR TITLE
Configurable Origins and small configuration fixes

### PR DESCRIPTION
**Pull request by djoelz**
This makes it configurable to specify multiple origins as allowed (default only local origin is allowed). Wildcard origin will not be supported as it is a security vulnerability.
It adds a compatibility check in configuration for windows paths.
Upgrades servlet config to add httponly and secure which will secure session cookies if used.

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Thanks for the fix! Tested and Looks good to me.

File: zeppelin-server/src/test/java/org/apache/zeppelin/security/SecurityUtilsTests.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import org.apache.zeppelin.utils.SecurityUtils;
+import org.junit.Test;
+
+import java.net.URISyntaxException;
+import java.net.UnknownHostException;
+
+/**
+ * Created by joelz on 8/19/15.
+ */
+public class SecurityUtilsTests {
```
Comment (jonbuffington) [inline zeppelin-server/src/test/java/org/apache/zeppelin/security/SecurityUtilsTests.java]: Do these tests run using the mvn test goal? I think the test class names should not be pluralized (e.g., SecurityUtilsTests -> SecurityUtilsTest). I may be wrong.

**Comment by jonbuffington** [comment]
@djoelz Thanks for adding the origin names a configuration option. Much appreciated.

File: zeppelin-server/src/test/java/org/apache/zeppelin/security/SecurityUtilsTests.java (hunk 1 above)
Comment (djoelz) [inline zeppelin-server/src/test/java/org/apache/zeppelin/security/SecurityUtilsTests.java]: You know more than me on this one. I can go and rename all my tests if this is true. Let me look at the build log to see if they ran.

Comment (djoelz) [inline zeppelin-server/src/test/java/org/apache/zeppelin/security/SecurityUtilsTests.java]: I will just rename to be consistent

**Comment by Leemoonsoo** [comment]
Thanks @djoelz. I'm merging it.

**Pull request closed without merging**